src/Fixer/Phpdoc/PhpdocVarWithoutNameFixer.php

```diff
@@ -68,21 +68,30 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens)
                 continue;
             }
 
-            $doc = new DocBlock($token->getContent());
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+            $nextToken = isset($tokens[$nextIndex]) ? $tokens[$nextIndex] : null;
 
-            // don't process single line docblocks
-            if (1 === count($doc->getLines())) {
-                continue;
+            // For people writing static public $foo instead of public static $foo
```

Comment:
Any tests to demonstrate this behaviour? Only seems to be tests for `public` properties atm, could do with ones for `static`, `private`, `protected` and `var`